ngraph/core/src/node.cpp

```diff
@@ -782,10 +820,16 @@ bool ov::Node::constant_fold(OutputVector& output_values, const OutputVector& in
         auto tensor = make_shared<HostTensor>(output.get_element_type(), output.get_partial_shape());
         output_tensors.push_back(tensor);
     }
+
+    ov::runtime::TensorVector input_runtime_tensors = create_tmp_tensors(input_tensors);
```

Comment:
You don't need create temporary buffers. Please create `runtime::TesorVector` instead of `ngraph::HostTensorVector` and call new Evaluate here.